schema/convert_json.go

```diff
@@ -181,12 +181,30 @@ func bodySchemaForCtyObjectType(typ cty.Type) *schema.BodySchema {
 	ret := &schema.BodySchema{
 		Attributes: make(map[string]*schema.AttributeSchema, len(attrTypes)),
 	}
+	blocks := make(map[string]*schema.BlockSchema, 0)
+
 	for name, attrType := range attrTypes {
 		ret.Attributes[name] = &schema.AttributeSchema{
 			Constraint: convertAttributeTypeToConstraint(attrType),
 			IsOptional: true,
 		}
+
+		if typeCanBeBlocks(attrType) {
+			fAttr := tfjson.SchemaAttribute{
+				AttributeType: attrType,
+			}
+			blockSchema, ok := blockSchemaForAttribute(&fAttr)
```

Comment:
It feels a little awkward to be creating the raw JSON types here to satisfy the interface but I cannot think of a cleaner solution without some larger refactoring 😅

Reply:
I agree, but came to the same conclusion